pkgs/development/python-modules/globre/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, pythonAtLeast
+, buildPythonPackage
+, fetchPypi
+, nose
+, coverage
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
You shouldn't have to set `name` like this manually.
```suggestion
```

Reply:
:+1: